qcodes/instrument_drivers/rigol/DS4000.py

```diff
@@ -250,3 +254,11 @@ def __init__(self, name, address, timeout=20, **kwargs):
 
         channels.lock()
         self.add_submodule('channels', channels)
+
+    def _check_firmware_version(self):
+        #Require version 00.02.03
+
+        idn = self.get_idn()
+        ver = LooseVersion(idn['firmware'])
+        if ver < LooseVersion('00.02.03'):
+            warnings.warn('Firmware version should be at least 00.02.03, data transfert may not work correctly')
```

Comment:
typo: `transfer` (without `t`) :) fix it and then I'll merge

Reply:
@YakBizzarro

Reply:
The line is also slightly too long. If we really want to get nitty-gritty (and we do!)

Reply:
Ok, done